tests/resources/inputs/calibration/CalibrationRecord_v0001.json

```diff
@@ -384,260 +384,261 @@
-          },
-          {
-            "groupID": 5,
-            "isMasked": false,
-            "twoTheta": 0.8690010092470938,
-            "dResolution": {
-              "minimum": 0.831438096217031,
-              "maximum": 4.157190481085156
-            },
-            "dRelativeResolution": 0.007548513084996265
-          }
+      "detectorState": {
+        "arc": [
+          -65.3,
+          104.95
+        ],
+        "wav": 2.1,
+        "freq": 60,
+        "guideStat": 1,
+        "lin": [
+          0.045,
+          0.043
         ]
       },
-      {
-        "focusGroup": {
-          "name": "Bank",
-          "definition": ""
-        },
-        "timeOfFlight": {
-          "minimum": 2742.6,
-          "maximum": 13713,
-          "binWidth": 1,
-          "binningMode": 1
-        },
-        "pixelGroupingParameters": [
-          {
-            "groupID": 0,
-            "isMasked": false,
-            "twoTheta": 1.8230747956560218,
-            "dResolution": {
-              "minimum": 0.4427877783643468,
-              "maximum": 2.2139388918217344
-            },
-            "dRelativeResolution": 0.003379514789208845
-          },
-          {
-            "groupID": 1,
-            "isMasked": false,
-            "twoTheta": 1.1545587481738246,
-            "dResolution": {
-              "minimum": 0.641323731469231,
-              "maximum": 3.2066186573461555
-            },
-            "dRelativeResolution": 0.005910630879382727
-          }
+      "gsasParameters": {
+        "alpha": 0.1,
+        "beta": [
+          0.02,
+          0.05
         ]
       },
-      {
-        "focusGroup": {
-          "name": "All",
-          "definition": ""
+      "particleBounds": {
+        "wavelength": {
+          "minimum": 0.7,
+          "maximum": 3.5
         },
-        "timeOfFlight": {
+        "tof": {
           "minimum": 2742.6,
-          "maximum": 13713,
-          "binWidth": 1,
-          "binningMode": 1
-        },
-        "pixelGroupingParameters": [
-          {
-            "groupID": 0,
-            "isMasked": false,
-            "twoTheta": 1.4888167719149363,
-            "dResolution": {
-              "minimum": 0.5165770976155033,
-              "maximum": 2.5828854880775167
-            },
-            "dRelativeResolution": 0.004814388725622084
-          }
-        ]
+          "maximum": 13713
+        }
+      },
+      "defaultGroupingSliceValue": 5,
+      "fwhmMultipliers": {
+        "minimum": 1.3,
+        "maximum": 1.3
       }
-    ],
-    "focusGroupCalibrationMetrics":{
-      "focusGroupName": "Column",
-      "calibrationMetric": [
-        {
-          "sigmaAverage": 0.002517952998136214,
-          "sigmaStandardDeviation": 0.00000878421857231147,
-          "strainAverage": 0.1515216185900471,
-          "strainStandardDeviation": 0.013151225165758604,
-          "twoThetaAverage": 2.110894794556694
+    },
+    "seedRun": 57514,
+    "useLiteMode": true,
```

Comment:
This is the only line I actually changed.  The rest is simply fixing the indentation.